src/components/Item/VisualizationItem/ItemHeaderButtons.js

```diff
@@ -54,16 +60,16 @@ const ItemHeaderButtons = (props, context) => {
 
     const itemHasMapView = () => hasMapView(item.type)
 
-    const handleMenuClick = (_, event) => setAnchorEl(event.currentTarget)
+    const handleMenuClick = () => setMenuIsOpen(true)
```

Comment:
I'm being picky here, but perhaps rename this to `handleButtonClick` or similar, it's not a click on the Menu.
Or if this becomes a toggle, `toggleMenu` could work.